src/commands/setup-autocomplete.ts

```diff
@@ -10,7 +10,7 @@ import { out } from "../util/interaction";
 const debug = require("debug")("mobile-center-cli:commands:setup-autocomplete");
 import { inspect } from "util";
 
-@help("Setup tab completion for your shell")
+@help("Set up autocompletion for the shell")
```

Comment:
These changes are unrelated, please send them in a separate PR.

Reply:
Yep, agreed, that was accidentally picked up from this PR: https://github.com/Microsoft/mobile-center-cli/pull/250, I'll revert it. Please let me know if these changes ever needed so I can create another PR for this.